RELEASE-NOTES.md

```diff
@@ -0,0 +1,16 @@
+# Release Notes
+
+## [0.9.0](https://github.com/redhat-ai-dev/ai-lab-samples/commits/v0.9.0) (2024-10-29)
+
+Initial release of AI Software Templates.
+
+### New Features
```

Comment:
Since this is our first official release should chatbot and codegen also be included in the list of features?

Reply:
Done.